From 5a6410aa1b86205ee7e1d2e129520f0f38e9eb59 Mon Sep 17 00:00:00 2001
From: Shenyufan <751890223@qq.com>
Date: Sun, 11 Aug 2024 12:35:58 +0800
Subject: [PATCH] fix-path

---
 CMakeLists.txt | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 00cfc9f..31a9ca3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -87,19 +87,19 @@ if(NOT WIN32)
         set(DLT_LIBRARY_INSTALLATION_PATH "${CMAKE_INSTALL_FULL_LIBDIR}")
     else()
         if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
-            set(CMAKE_INSTALL_PREFIX "DLTViewer")
+            set(CMAKE_INSTALL_PREFIX)
         endif()
         if(NOT DLT_PLUGIN_INSTALLATION_PATH)
-            set(DLT_PLUGIN_INSTALLATION_PATH "${DLT_APP_DIR_NAME}/usr/bin/plugins")
+            set(DLT_PLUGIN_INSTALLATION_PATH "${CMAKE_INSTALL_PREFIX}/bin/plugins")
         endif()
         if(NOT DLT_RESOURCE_INSTALLATION_PATH)
-            set(DLT_RESOURCE_INSTALLATION_PATH "${DLT_APP_DIR_NAME}/usr/share")
+            set(DLT_RESOURCE_INSTALLATION_PATH "${CMAKE_INSTALL_PREFIX}/share")
         endif()
         if(NOT DLT_EXECUTABLE_INSTALLATION_PATH)
-            set(DLT_EXECUTABLE_INSTALLATION_PATH "${DLT_APP_DIR_NAME}/usr/bin")
+            set(DLT_EXECUTABLE_INSTALLATION_PATH "${CMAKE_INSTALL_PREFIX}/bin")
         endif()
         if(NOT DLT_LIBRARY_INSTALLATION_PATH)
-            set(DLT_LIBRARY_INSTALLATION_PATH "${DLT_APP_DIR_NAME}/usr/lib")
+            set(DLT_LIBRARY_INSTALLATION_PATH "${CMAKE_INSTALL_PREFIX}/lib")
         endif()
     endif()
 else()
-- 
2.43.4

